geometry/eolearn/geometry/transformations.py

```diff
@@ -26,11 +26,12 @@
 
 from sentinelhub import CRS, bbox_to_dimensions
 from eolearn.core import EOTask, FeatureType, FeatureTypeSet
+from eolearn.core.utilities import renamed_and_deprecated
 
 LOGGER = logging.getLogger(__name__)
 
 
-class VectorToRaster(EOTask):
+class VectorToRasterTask(EOTask):
```

Comment:
Ah, I see. I'll take a closer look and decide whether it makes sense to keep them static or not. If there is a lot of interaction then keeping them static really could cause issues with inheritance.

Reply:
the method `_parse_main_params` wont be static anymore. As for `_is_geopandas_object` and `_vector_is_timeless`, I would like to make them regular functions, since they perform simple checks on objects that have nothing to do with the tasks themselves. I also don't see why they would be overridden.